mrmustard/lab_dev/samplers.py

```diff
@@ -229,7 +229,7 @@ def sample(self, state: State, n_samples: int = 1000, seed: int | None = None) -
         for unique_sample, counts in zip(unique_samples, counts):
             quad = np.array([[unique_sample] + [None] * (state.n_modes - 1)])
             quad = quad if isinstance(state, Ket) else math.tile(quad, (1, 2))
-            reduced_rep = (state >> BtoQ([initial_mode], phi=self._phi)).representation(quad)
+            reduced_rep = (state >> BtoQ([initial_mode], phi=self._phi)).ansatz(quad)
```

Comment:
I know it doesn't change what's happening, but maybe update the name to reduced_ansatz